CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 -   Update unit tests to use userEvent instead of fireEvent
 -   Move all form items under a `forms` folder
 -   Upgrade storybook to v8
+-   Remove Radix props dependency
```

Comment:
This should go under the `unreleased` header when you update your branch with the latest changes on `main`

Reply:
@rcaragon The entry is duplicate, it needs to be removed here

Reply:
✅ https://github.com/aragon/ods/pull/218/commits/5049d11d19e55f22bd62f2096fd8f18a28ee5c45